util/find-doc-nits

```diff
@@ -444,6 +444,21 @@ sub check {
         check_section_location($id, $contents, "EXAMPLES", "SEE ALSO");
     }
 
+    # Check for proper in-man-3 API links.
+    while ( $contents =~ /L<([^>]*)>/g ) {
+        my $target = $1;
+        next if $target =~ /^https?:/;
+        next if $target =~ m@^/@;
+        next if $target =~ m@\([1357]\)@;
+        next if $target =~ m@\([1357]\)/[^>]*>@;
+        err($id, "Section missing in L<$target>")
+    }
+    while ( $contents =~ /L<([^>]*)\(3\)(?:\/.*)?>/g ) {
+        my $target = $1;
+        err($id, "Bad L<$target>")
+            unless $target =~ /^[_[:alpha:]][_[:alnum:]]*$/
+    }
```

Comment:
More generally speaking, doesn't this mean that we simply do not allow links that doesn't match any of these regexps?

`m,L<https?://,`
`m,L</,`
`m,L<(?:.*?|)(.*?)\((\d)\)(?:/.*?)?>`

with the added caveat that, for the last one, we need to check if `$2 == 3`, and if it is, that `$1` has valid symbol syntax.

Reply:
I don't know.  I do know that what is there works now.